components/project-card/meta/title.jsx

```diff
@@ -13,18 +13,31 @@ class Title extends React.Component {
   }
 
   render() {
-    let classnames = classNames(`title mb-0 h4-heading`, this.props.className);
+    let content;
     let title = this.props.title;
+    let view = this.props.view;
```

Comment:
1. Change this prop from `view` to `simpleView`.
2. Remove changes made to `project-card-detailed.jsx` that were related to this props
3. In `project-card-simple.jsx`
   1. Remove `simpleView` state
   2. Change `view={this.state.simpleView}` for `<Title>` to `simpleView={true}`